### PR TITLE
Add command `OpenShift: Show Output Channel`

### LINKED ISSUE #645: Add command `OpenShift: Show Output Channel`
Add 'Show Output Channel' command to OpenShift View Title and to command palette. 
It should show Output Channels view and activate OpenShift channel.

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix #645.

### CODE REVIEW

**Comment by sudhirverma** [comment]
testPR

**Comment by sudhirverma** [comment]
@dgolovin review this pr

File: src/cli.ts [hunk 1]
```diff
@@ -52,15 +52,23 @@ export interface OdoChannel {
     print(text: string): void;
 }
 
+export const openShiftOutput = {
```
Comment (dgolovin) [inline src/cli.ts]: No need to create new object with map. There is always one OpenShift output channel. Just add this method to Cli:

``` JavaScript
showOutputChannel() {
   this.odoChannel.show();
}

File: src/openshift/cluster.ts [hunk 2]
```diff
@@ -42,6 +42,13 @@ export class Cluster extends OpenShiftItem {
         Cluster.odo.executeInTerminal(Command.printOdoVersion());
     }
 
+    static async showOpenShiftOutput() {
```
Comment (dgolovin) [inline src/openshift/cluster.ts]: should be just:

``` JavaScript 
Cli.getInstance().showOutputChannel()
```

**Pull request merged**